packages/js/src/dashboard/widgets/widget.js

```diff
@@ -16,31 +16,64 @@ export const WidgetTitle = ( { children, ...props } ) => (
 WidgetTitle.displayName = "Widget.Title";
 
 /**
- * @param {ReactNode} children The content of the paragraph.
- * @param {string} [learnMoreLink] The URL to the learn more page.
- * @returns {JSX.Element} The widget tooltip with paragraph and optional learn more link.
+ * @param {ReactNode} content The content of the tooltip.
+ * @param {ReactNode} children The rest of the tooltip.
+ * @returns {JSX.Element} The widget tooltip.
  */
-export const WidgetTooltip = ( { children, learnMoreLink } ) => (
+export const WidgetTooltip = ( { content, children } ) => (
 	<InfoTooltip>
-		<p className="yst-mb-2">{ children }</p>
-		{ learnMoreLink && <LearnMoreLink href={ learnMoreLink } /> }
+		<p className="yst-mb-2 yst-text-slate-600">{ content }</p>
+		{ children }
 	</InfoTooltip>
 );
 WidgetTooltip.displayName = "Widget.Tooltip";
 
+/**
+ * @param {Object[]} [dataSources] The sources of the data in the widget.
+ * @returns {JSX.Element} The sources of the data in the widget as a component.
+ */
+export const WidgetDataSources = ( { dataSources } ) => (
+	<div className="yst-leading-4 yst-border-t yst-mt-3 yst-border-slate-200 yst-italic yst-text-[11px]">
```

Comment:
This `leading` requirement here makes me think.
I messed up with the `leading-[0]` in the InfoTooltip, we should reset it on the `TooltipWithContext` wrapper instead.
The reason you don't see it for the paragraph is due to WP styling.
I suggest removing the `yst-leading-4` here, in favor of adding it there.

Reply:
[Fixed](https://github.com/Yoast/wordpress-seo/pull/22103/commits/1ba93349d4bac595d8231fb663fa62388c770078).